src/splitInteger.test.js

```diff
@@ -4,18 +4,22 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(6, 2))
+    .toStrictEqual([3, 3]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(8, 1))
+    .toStrictEqual([8]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  expect(splitInteger(17, 4))
+    .toStrictEqual([4, 4, 4, 5]);
 });
 
 test('should add zeros if value < numberOfParts', () => {
-
-});
+  expect(splitInteger(2, 4))
+    .toStrictEqual([0, 0, 1, 1]);
+});
\ No newline at end of file
```

Comment:
No newline at the end of file. It's a common convention to put a newline at the end of a file. It's not going to break anything, but it's always better to follow the convention.

Reply:
fix